src/wstool/multiproject_cli.py

```diff
@@ -1327,9 +1327,12 @@ def cmd_scrape(self, target_path, argv, config=None):
             usage=usage,
             description=__MULTIPRO_CMD_DICT__["scrape"],
             epilog="See: http://www.ros.org/wiki/rosinstall for details\n")
-        parser.add_option("-y", "--confirm", dest="confirm", default='',
+        parser.add_option("-y", "--confirm", dest="confirm", default=False,
```

Comment:
Could you please also update the bash completions if adding command-line options: https://github.com/vcstools/wstool/blob/master/completion/wstool-completion.bash

Reply:
`scrape` wasn't in the bash completion list before, but I added it with all its options now :wink:

Reply:
# :raised_hands: